.github/ISSUE_TEMPLATE/release.md

```diff
@@ -29,8 +29,11 @@ examples of each step.
   - [ ] Run the [Tag workflow][tag-workflow] on the `master` branch with the next release candidate tag.
 -->
 - [ ] Updated all version information in the documentation on the relevant release branch.
+<!-- Uncomment the following block only if cutting a minor release. -->
+<!--
   - [ ] Update the [`version` front-matter](https://github.com/crossplane/crossplane/blob/master/docs/_index.md?plain=1#L8) in docs/_index.md
   - [ ] Update the [`alias` front-matter](https://github.com/crossplane/crossplane/blob/master/docs/_index.md?plain=1#L6) in docs/_index.md
+-->
```

Comment:
Nit: Is there any way to refactor this to avoid adding another "uncomment this if" block? e.g. Could we move the doc version updates down to the end of the list such that they "flow into" the existing "uncomment this" block down there?

Reply:
We could, but we typically want the docs updates to happen prior to the release, and moving them to the bottom would likely result in two separate PRs. I wonder if it would make sense just to have separate templates for each? I'd actually like to move us over to the new [forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) soon as well.

Reply:
Yeah separate templates sounds smoother to me.